docs/custom_theme/main.html

```diff
@@ -0,0 +1,38 @@
+<!-- Elements added to main will be displayed on all pages -->
```

Comment:
I'm not sure if I got this one per se, as I think it might just be the general theme? But can see a few screenshots in the [PR description](https://github.com/planetlabs/planet-client-python/pull/747#issue-1441410759) now.